build

```diff
@@ -128,6 +128,16 @@ def compile_docker(config, target, src_dir, tgt_dir, debug=False):
     dshell('/tgt/app', '/tgt/qt/bin/qmake /src/wkhtmltopdf.pro CONFIG+=silent')
     dshell('/tgt/app', 'make install INSTALL_ROOT=/tgt/wkhtmltox')
 
+    aftercompile = config['docker-targets'][target].get('after_compile', '')
+    if aftercompile:
+        script = 'after_compile/%s.sh' % aftercompile
+        if not os.path.exists(os.path.abspath(script)):
```

Comment:
Let's remove this check, as it won't ever be faced by users.

Reply:
I guess it would be easier to debug issues. Without this check, in case you made a typo in the `after_compile` property, the build process will silently finish. What do you think?

Reply:
Yeah, but the output would be checked anyway -- this is a configuration level issue, so would be caught while a PR is being developed.

Reply:
done